tests/HL_testing_input.json

```diff
@@ -0,0 +1,17 @@
+{
+    "model_name" : ["gpt2","gpt2-medium","TinyLlama/TinyLlama-1.1B-Chat-v1.0","Salesforce/codegen-350M-mono","wtang06/mpt-125m-c4"],
+    "num_cores" : [8],

```

Comment:
Why 8?

Reply:
We can add more numbers in the range of 1 to number free cores, such as 8,16